### PR TITLE
Alerting docs: adds silence RBAC 11.1

### PR DESCRIPTION
**Pull request by brendamuir**
adds docs for rbac silences 11.1 features

### CODE REVIEW

**Comment by brendamuir** [comment]
Hi Matt - was using your suggestions in: https://docs.google.com/document/d/1-44V-Cz51wuG_U68IjqtzLXBUN6dnVXWOlaZcOSSXIQ/edit. 

I think I updated for Rule-specific silences with permissions. I am confused with the RBAC -- because I see the permissions already listen in the doc here https://grafana.com/docs/grafana/next/alerting/set-up/configure-rbac/access-roles/.

Can you take a look and add anything to this pr that needs updating? I have to admit to not really understanding the backwards compatibility section!

**Comment by diegommm** [comment]
Hi! Greetings from the Release Guild. Just letting you know I'm updating your PR's milestone to [11.2.x](https://github.com/grafana/grafana/milestone/529). If you need to see your changes in a previous version, please consider adding a backport label.

Thank you!

**Comment by brendamuir** [comment]
hey @JacobsonMT just a gentle reminder :) thanks!

**Comment by brendamuir** [comment]
Hey @JacobsonMT sorry to be a nag. With the What's New going out soon and already out for Cloud - we may get questions where the docs are. Would you rather meet and then I can add docs/ or add some draft docs?

**Comment by JacobsonMT** [comment]
Sorry for the delay! 

- Reworded some definitions for clarity
- Reworked the OSS `Manage access using folders` and Enterprise `Manage access using folders or data sources` sections to be a bit more specific on what can be done. For instance, using "extend" access instead of "customize" , simplifying the table to show just the additional access provided by the `Folder Permission` and including some examples in the Enterprise case.
- Add a section defining what we mean by `rule-specific silence` in the `Configure silences` page.

@yuri-tceretian can you double check my changes / examples?

File: docs/sources/alerting/set-up/configure-rbac/access-folders/index.md [hunk 1]
```diff
@@ -18,21 +18,36 @@ weight: 200
...
 
-Details of how role access can combine with folder permissions for Grafana Alerting are below.
+This allows different users, teams, or service accounts to have customized access to modify or silence alert rules in specific folders or using specific data sources.
+
+Details on the additional access provided by folder permissions are below.
+
+| Folder permission | Additional Access                                                                                                                             |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| View              | Read access to all general silences. Read access to alert rules and their rule-specific silences **only** in the given folder and subfolders. |
+| Edit              | View access + write access to alert rules and their rule-specific silences **only** in the given folder and subfolders.                       |
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-rbac/access-folders/index.md:34]:
```suggestion
| Edit              | View access and write access to alert rules and their rule-specific silences _only_ in the given folder and subfolders.                   |
```

File: docs/sources/alerting/set-up/configure-rbac/access-folders/index.md [hunk 2]
```diff
@@ -18,21 +18,36 @@ weight: 200
...
-| Viewer | Admin  | Write access to alert rules **only** in the folders where the Admin permission is added. |
-| Viewer | Edit   | Write access to alert rules **only** in the folders where the Edit permission is added.  |
-| Viewer | View   | Read access to alert rules in all folders.                                               |
+| Role          | Folder permission | Access                                                                                                                                                                   |
+| ------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| No Basic Role | -                 | None. No access to alert rules or rule-specific silences in the folder.                                                                                                  |
+| No Basic Role | View              | Read access to all general silences. Read access to alert rules and their rule-specific silences **only** in the given folder and subfolders.                            |
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-rbac/access-folders/index.md:42]:
```suggestion
| No Basic Role | View              | Read access to all general silences. Read access to alert rules and their rule-specific silences _only_ in the given folder and subfolders.                              |
```

File: docs/sources/alerting/set-up/configure-rbac/access-folders/index.md [hunk 3]
```diff
@@ -18,21 +18,36 @@ weight: 200
...
+| No Basic Role | Edit              | Above access + write access to alert rules (given necessary data source query permissions) and their rule-specific silences **only** in the given folder and subfolders. |
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-rbac/access-folders/index.md:43]:
```suggestion
| No Basic Role | Edit              | Above access and write access to alert rules (given necessary data source query permissions) and their rule-specific silences _only_ in the given folder and subfolders. |
```

File: docs/sources/alerting/set-up/configure-rbac/access-folders/index.md [hunk 4]
```diff
@@ -18,21 +18,36 @@ weight: 200
...
+| Viewer        | -                 | Read access to alert rules and silences in all folders.                                                                                                                  |
+| Viewer        | Edit              | Read access to alert rules and silences in all folders. Write access to alert rules and their rule-specific silences **only** in the given folder and subfolders.        |
+| Editor        | View              | Read+write access to alert rules and silences in all folders. (Access can only be extended.)                                                                             |
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-rbac/access-folders/index.md:46]:
```suggestion
| Editor        | View              | Read and write access to alert rules and silences in all folders. Access can only be extended.                                                                             |
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 5]
```diff
@@ -42,19 +42,17 @@ To assign roles, admins need to complete the following steps.
...
 ## Manage access using folder permissions
 
-You can further customize access for alert rules by assigning permissions to individual folders.
+You can extend the access provided by a role to alert rules and rule-specific silences by assigning permissions to individual folders.
 
-This prevents every user from having access to modify all alert rules and gives them access to the folders with the alert rules they're working on.
+This allows different users, teams, or service accounts to have customized access to modify or silence alert rules in specific folders.
 
-Details on the adding folder permissions as well as roles and the access that provides for Grafana Alerting is below.
+Details on the additional access provided by folder permissions is below.
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-roles/index.md:49]: https://developers.google.com/style/word-list#below
```suggestion
Refer to the following table for details on the additional access provided by folder permissions:
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 6]
```diff
@@ -42,19 +42,17 @@ To assign roles, admins need to complete the following steps.
...
-| Editor | -                 | Write access to alert rules in all folders.                                                                                          |
-| Viewer | Admin             | Read access to alert rules in all folders. Write access to alert rules **only** in the folders where the Admin permission is added.  |
-| Viewer | Editor            | Read access to alert rules in all folders. Write access to alert rules **only** in the folders where the Editor permission is added. |
-| Viewer | Viewer            | Read access to alert rules in all folders.                                                                                           |
+| Folder permission | Additional Access                                                                                         |
+| ----------------- | --------------------------------------------------------------------------------------------------------- |
+| View              | No additional access: all permissions already contained in Viewer role.                                   |
+| Edit              | Write access to alert rules and their rule-specific silences **only** in the given folder and subfolders. |
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-roles/index.md:54]: Again, this one will break prettier

```suggestion
| Edit              | Write access to alert rules and their rule-specific silences _only_ in the given folder and subfolders. |
```

File: docs/sources/alerting/set-up/configure-rbac/access-folders/index.md [hunk 7]
```diff
@@ -18,21 +18,36 @@ weight: 200
...
 {{< /admonition >}}
 
-Details of how role access can combine with folder permissions for Grafana Alerting are below.
+This allows different users, teams, or service accounts to have customized access to modify or silence alert rules in specific folders or using specific data sources.
+
+Details on the additional access provided by folder permissions are below.
+
+| Folder permission | Additional Access                                                                                                                             |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| View              | Read access to all general silences. Read access to alert rules and their rule-specific silences **only** in the given folder and subfolders. |
```
[the remark this answers is not in the document]
Comment (JacobsonMT) [reply docs/sources/alerting/set-up/configure-rbac/access-folders/index.md]: If you mean that even the `_only_` won't work, we can just get rid of the emphasis altogether. It's not really necessary anyways.

[the remark this answers is not in the document]
Comment (jdbaldry) [reply docs/sources/alerting/set-up/configure-rbac/access-folders/index.md]: Oh wait, I can provide a suggestion that won't break prettier when applied, I'll update it.

File: docs/sources/alerting/set-up/configure-rbac/access-folders/index.md [hunk 8]
```diff
@@ -18,21 +18,36 @@ weight: 200
...
+| No Basic Role | -                 | None. No access to alert rules or rule-specific silences in the folder.                                                                                                  |
+| No Basic Role | View              | Read access to all general silences. Read access to alert rules and their rule-specific silences _only_ in the given folder and subfolders.                              |
+| No Basic Role | Edit              | Above access and write access to alert rules (given necessary data source query permissions) and their rule-specific silences _only_ in the given folder and subfolders. |
+| Viewer        | -                 | Read access to alert rules and silences in all folders.                                                                                                                  |
+| Viewer        | Edit              | Read access to alert rules and silences in all folders. Write access to alert rules and their rule-specific silences **only** in the given folder and subfolders.        |
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/configure-rbac/access-folders/index.md:45]:
```suggestion
| Viewer        | Edit              | Read access to alert rules and silences in all folders. Write access to alert rules and their rule-specific silences _only_ in the given folder and subfolders.          |
```

**Review by jdbaldry — COMMENTED** [review]
Just one more

**Review by jdbaldry — APPROVED** [review]
LGTM

**Pull request merged**